src/yupSchema.js

```diff
@@ -101,6 +101,13 @@ const yupSchemas = {
   date: ({ minDate, maxDate }) => {
     let dateString = string()
       .nullable()
+      .transform((value) => {
+        if (value === '') {
```

Comment:
Please leave a comment about why this is needed, similar to how we did in the radio. It can be as simple as

```js
// @BUG RMT-518 - Same reason to radioOrSelect above.
```